.github/workflows/demo-integrations.yml

```diff
@@ -70,26 +69,21 @@ jobs:
         run: |
           npm run serve:compiled & sleep 5
           curl -X GET -I -f "http://localhost:3333"
-
       - name: Run screenshot tests on demo/localhost
-        run: |
-          npm run concurrently -- \
-                             "npm run cy2:run:localhost" \
-                             "npm run cy2:run:localhost"
+        run: npm run concurrently "npm run cy2:run:localhost" "npm run cy2:run:localhost"
 
       - name: Run screenshot tests on demo/next
         continue-on-error: true
-        run: |
-          npm run concurrently -- \
-                         "npm run cy2:run:demo:next" \
-                         "npm run cy2:run:demo:next"
+        env:
+          CYPRESS_failOnSnapshotDiff: true # collect all diff files for lumberjack bot
```

Comment:
there is no mistake? seems like it should be false

Reply:
fixed